### ISSUE TITLE
ModuleNotFoundError: No module named 'arcade'

### ISSUE THREAD

**Issue by nb43** [pythonarcade/arcade#1903]
## Bug Report
Its my first time trying to run arcade. I did it using VSCode in a non-virtual environment.
(Would rather not use PyCharm or venv unless they are requirements?)

## System Info
Arcade 2.6.17
vendor: Apple
renderer: Apple M2
version: (4, 1)
python: 3.11.5 (main, Aug 24 2023, 15:09:45) [Clang 14.0.3 (clang-1403.0.22.14.1)]
platform: darwin

### Actual behavior:
```
Traceback (most recent call last):
  File "$PATH/arcade_platformer/arcade_platformer/main.py", line 5, in <module>
    import arcade
ModuleNotFoundError: No module named 'arcade'
```

### Expected behavior:
That the code would run, since arcade is installed.

## Documentation request:
Installation guide for getting Python Arcade to work on VSCode on Apple silicon. Perhaps a troubleshooting guide. Perhaps clarification whether venv and/or PyCharm are required for arcade to work.

**Comment by einarf** [comment]
You probably installed arcade in a different python interpreter than VSCode selected? Can you show the install log from `pip`, `pip list`, `python --version` and the python interpreter version showed in the lower left bar?

I don't think this is a arcade specific issue, but a general python/vscode one.

**Comment by nb43** [comment]
Thank you for taking the time to respond!

I installed arcade through terminal.
At first the `pip3` command did not work. And I know I had installed homebrew and python at an earlier point. But copying homebrew from .bash to .zsh solved that. After that I installed arcade with `pip3 install arcade`.
Perhaps arcade is stuck to .bash? Or I need to repeat if from terminal within VSCode?

Anyways, here is what you requested:

`pip`:
zsh: command not found: pip

`pip3 list`:
Package           Version
arcade            2.6.17
attrs             23.1.0
cffi              1.15.1
Pillow            9.3.0
pip               23.2.1
pycparser         2.21
pyglet            2.0.dev23
pymunk            6.4.0
pytiled-parser    2.2.0
setuptools        68.1.2
typing_extensions 4.7.1
wheel             0.41.2

`python3 --version`:
Python 3.11.5
